### PR TITLE
Add support adding/removing members and transferring leadership via REST API. Fixes #1544 and fixes #1543

### LINKED ISSUE #1543: Support add/remove raft peer via REST
Note that REST is not available until after the cluster is bootstrapped, so this won't work for all cases

### LINKED ISSUE #1544: Support transfer raft leadership via REST

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

**Comment by andrewpmartinez** [comment]
Fixes #1544 and fixes #1543

File: controller/apierror/messages.go [hunk 1]
```diff
@@ -198,4 +198,12 @@ const (
 	ServerTooManyRequestsCode    string = "SERVER_TOO_MANY_REQUESTS"
 	ServerTooManyRequestsMessage string = "Too many requests to alter state have been issued. Please slow your request rate or try again later."
 	ServerTooManyRequestsStatus  int    = http.StatusTooManyRequests
+
+	ServerNotRunningInClusterModeCode    string = "SERVER_NOT_RUNNING_IN_CLUSTER_MODE"
```
Comment (andrewpmartinez) [inline controller/apierror/messages.go:202]: Cluster mode? HA mode? I have a capability in the edge section called `HA_CONTROLLER`.

I think we should try to be consistent.

Comment (plorenz) [reply controller/apierror/messages.go:202]: Makes sense, I'll change it to HA

Comment (plorenz) [reply controller/apierror/messages.go]: Done

**Pull request merged**